sdks/common/runner-api/src/main/proto/beam_runner_api.proto

```diff
@@ -525,6 +525,11 @@ message Trigger {
   message Never {
   }
 
+  // Always ready. This can also be expressed as ElementCount(1) but
+  // is more explicit.
+  message Always {
```

Comment:
Thinking out loud -- should we rename this to something like `Anytime`?

(This may be putting the change in the wrong place, but) "Triggering.Always" makes me think it will always trigger; as opposed to "always being _ready_" to trigger.